lib/manager/kustomize/types.ts

```diff
@@ -4,8 +4,16 @@ export interface Image {
   newName?: string;
   digest?: string;
 }
+
+export interface HelmChart {
+  name: string;
+  repo: string;
+  version: string;
+}
+
 export interface Kustomize {
   kind: string;
   bases: string[];
   images: Image[];
+  helmCharts: HelmChart[];
```

Comment:
```suggestion
  helmCharts?: HelmChart[];
```

Fully optional

Reply:
In that case, shouldn't bases and images be optional too? Neither are required in a kustomization file.

Reply:
Sure, but should be separate pr